fhir-persistence-schema/src/main/resources/add_resource_template.sql

```diff
@@ -14,44 +14,26 @@
 -- p_payload:    the BLOB (of JSON) which is the resource content
 -- p_last_updated the last_updated time given by the FHIR server
 -- p_is_deleted: the soft delete flag
--- p_tx_correlation_id: used to group multiple resources together in one transaction
--- p_changed_by: the identity of the user who made the change
--- p_correlation_token: the tracking token used for audit trail correlation
--- p_version_id: the version id if this is a replicated message
 -- p_json_version: the version FHIR injected into the JSON
 -- p_str_values        an array of string parameter values
 -- p_number_values     an array of number parameter values
 -- p_date_values       an array of date parameter values
 -- p_latlng_values     an array of lat/long parameter values
 -- p_token_values      an array of token values
 -- p_quantity_values   an array of quantity values
--- p_write_rep_log: flag indicating if a row should be written to the replication log table
 -- o_resource_id: output field returning the newly assigned resource_id value
 -- ----------------------------------------------------------------------------
     ( IN p_logical_id                    VARCHAR(255 OCTETS), 
       IN p_payload                          BLOB(2147483647),
       IN p_last_updated                TIMESTAMP,
       IN p_is_deleted                       CHAR(1),
-      IN p_source_key                    VARCHAR(64),
-      IN p_tx_correlation_id             VARCHAR(36),
-      IN p_changed_by                    VARCHAR(64),
-      IN p_correlation_token             VARCHAR(36),
-      IN p_tenant_data_id                VARCHAR(36),
-      IN p_reason                        VARCHAR(255),
-      IN p_event                         VARCHAR(32),
-      IN p_site_id                       VARCHAR(255),
-      IN p_study_id                      VARCHAR(255),
-      IN p_service_id                    VARCHAR(32),
-      IN p_patient_id                    VARCHAR(255),
-      IN p_version                           INT,
```

Comment:
Are those dropped fields used for replication only?

Reply:
yes, that's right